packages/react-celo/src/utils/useProviders.ts

```diff
@@ -44,18 +45,23 @@ export function getRecent(): Maybe<Provider> {
 
 export default function useProviders(
   wallets: WalletEntry[] = [],
+  allScreens: Record<string, React.FC<ConnectorProps>>,
   sort = defaultProviderSort,
   search?: string
 ) {
   const record: Record<string, Provider> = useMemo(
     () => ({
-      ...PROVIDERS,
+      ...Object.keys(allScreens).reduce((all, current) => {
+        // @ts-ignore
```

Comment:
not the biggest fan, can we type this better?

Reply:
oh right. yeah had to go to appt and just wanted to get this up. 

probably can do better